### PR TITLE
Simsexcel

### PR DESCRIPTION
**Pull request by KIngSimeone**

### CODE REVIEW

File: routelift_api/Logistics/views.py [hunk 1]
```diff
@@ -222,8 +218,8 @@ def delete_company(request, company_id):
                                 "login.")
 
     resource = "company"
-    operation = "read"
-
+    operation = "delete"
+    print("at validate role")
```
Comment (DeeMATT) [inline routelift_api/Logistics/views.py:222]: I think this should be removed

**Review by DeeMATT — CHANGES_REQUESTED** [review]
user_test.xlsx file should not be pushed to the server. If I'm correct, I think this was meant for the test suite. If so, then we can discuss a way out

File: routelift_api/Logistics/views.py (hunk 1 above)
Comment (KIngSimeone) [reply routelift_api/Logistics/views.py:222]: Do u mean the role validator?

Comment (KIngSimeone) [reply routelift_api/Logistics/views.py:222]: or the print statement

Comment (KIngSimeone) [reply routelift_api/Logistics/views.py:222]: print was while debugging, will be removed

**Comment by KIngSimeone** [comment]
> user_test.xlsx file should not be pushed to the server. If I'm correct, I think this was meant for the test suite. If so, then we can discuss a way out

Removed Sir

**Review by DeeMATT — APPROVED** [review]

**Pull request merged**